Extensions/TextObject/textruntimeobject.js

```diff
@@ -261,6 +276,23 @@ gdjs.TextRuntimeObject.prototype.getColor = function(str) {
     return this._color[0] + ";" + this._color[1] + ";" + this._color[2];
 };
 
+/**
+ * Set the text alignment for multiline text objects.
+ * @param {String} alignment The text alignment.
```

Comment:
`param {string}`

String is a class "boxing" the string primitive type: https://stackoverflow.com/questions/17256182/what-is-the-difference-between-string-primitives-and-string-objects-in-javascrip

In short: never use String, Number or Boolean as types. Always the lowercase version.